dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -546,31 +522,17 @@ def enter_shell(**kwargs) -> RunCommandResult:
     * executes the command to drop the user to Breeze shell
 
     """
-    verbose = kwargs["verbose"]
-    dry_run = kwargs["dry_run"]
-    perform_environment_checks(verbose=verbose)
+    perform_environment_checks()
     if read_from_cache_file("suppress_asciiart") is None:
         get_console().print(ASCIIART, style=ASCIIART_STYLE)
     if read_from_cache_file("suppress_cheatsheet") is None:
         get_console().print(CHEATSHEET, style=CHEATSHEET_STYLE)
-    enter_shell_params = ShellParams(**filter_out_none(**kwargs))
-    rebuild_or_pull_ci_image_if_needed(command_params=enter_shell_params, dry_run=dry_run, verbose=verbose)
-    if enter_shell_params.include_mypy_volume:
+    shell_params = ShellParams(**filter_out_none(**kwargs))
+    rebuild_or_pull_ci_image_if_needed(
+        command_params=shell_params,
+    )
+    if shell_params.include_mypy_volume:
         create_mypy_volume_if_needed()
-    return run_shell(verbose, dry_run, enter_shell_params)
-
-
-def run_shell(verbose: bool, dry_run: bool, shell_params: ShellParams) -> RunCommandResult:
-    """
-    Executes a shell command built from params passed.
-    * prints information about the build
-    * constructs docker compose command to enter shell
-    * executes it
-
-    :param verbose: print commands when running
-    :param dry_run: do not execute "write" commands - just print what would happen
-    :param shell_params: parameters of the execution
-    """
```

Comment:
Nice, assuming this is the only place that method was used, this makes a lot more sense.

Reply:
Yep. It was added by 2 people I think or converted from Bash earlier so it's a bit of a "legacy remnant"